aries-site/src/pages/templates/forms.mdx

```diff
@@ -23,105 +41,73 @@ import { Example } from '../../layouts';
 
-### Using help text
+**Format and fields**
+- Repeat the field label to use as placeholder text 
+- Use generic labels on buttons, such as “Submit” 
 
-In most cases, the FormField label should be clear enough to instruct the user how to complete a field. However, if help text is required, ensure that it is kept brief and precise.
+**Actions after the user submits information**
+- Restrict resource naming. Allow spaces, punctuation and unicode characters (ex. Björk Guðmundsdóttir, 联想, John Q. Public, etc.)
 
-Help text may be relevant for cases such as [password requirements](#sign-up) or [additional resources](#customize) on a field's context.
 
-### Capitalization in forms
+## Single field forms
 
-Writing standards contribute to the voice, tone, and grammatical accuracy that inform the experiences we create. These standards are aligned with HPE Brand.
-
-Within a form, you should:
-
-- Use title case and capitalize short prepositions when they are the last word in a sentence (e.g. Sign Up, Sign Out)
-- For headings and captions, use sentence case
-
-### Button placement and labeling
-
-Submit, reset, or cancel buttons should always be placed at the bottom of a Form and be left-aligned.
-
-Button labels should be specific to the action of the button. For example, a [sign-up form](#sign-up) may have a submit button with label "Sign Up". Alternately, a [password change form](#change-password) may have a button with label "Update Password".
-
-Button label specificity provides users confidence in the action they are performing.
-
-### User-Entered Input
-
-There should be few restrictions on user-entered input.
-
-- Resource naming should allow for spaces, punctional characters, unicode characters.
-  - Examples: Björk Guðmundsdóttir, 联想, John Q. Public, etc.
-- User-entered input needs to be encoded to prevent [SQL injection](https://www.w3schools.com/sql/sql_injection.asp) and [XSS attacks](https://en.wikipedia.org/wiki/Cross-site_scripting).
-
-### Form submission
-
-Visual feedback is important to instill confidence in a user after form submission.
-
-**On submission, your form should:**
-
-- In the case of an error, focus should be placed on the input element where the error occurred.
-- Provide user with indication that form was submitted successfully (e.g. notification, success page, etc.)
-- Display all relevant errors accompanied by text explaining actions needed to resolve.
-- Allow user to continue navigating through the application immediately
-
-Never force the user to wait for post-submission processing to complete before they can continue using an application.
-
-### Accessibility
-
-All Form input elements need to be contained within a FormField.
+- For [single form fields](#single-required-formfield) (ex: delete confirmation request), use an asterisk to denote that it is a "required" field. This can avoid user confusion and reduce the chance of "required" input error.
 
-**FormFields need to have:**
+<ExampleImagePreview
```

Comment:
This example needs to be moved, it's not related to Single Field forms. Let me know if you think this quick link is helpful stilll and where you'd like to move it. I can move it for you.

Reply:
Has this been resolved? I looked for it and it seems we've worked it out of the document...?

Reply:
This is still here. Can you let me know if you'd like it deleted or moved somewhere else on the page?